### PR TITLE
adding download from huggingface when the finetune_output_dir is empty

### PR DESCRIPTION
**Pull request by simona-rc**

### CODE REVIEW

File: mammal/examples/dti_bindingdb_kd/main_infer.py [hunk 1]
```diff
@@ -55,17 +55,24 @@ def dti_bindingdb_kd_infer(
...
-    # load tokenizer
-    tokenizer_op = ModularTokenizerOp.from_pretrained(
-        os.path.join(finetune_output_dir, "tokenizer")
-    )
-
-    # Load model
-    nn_model = Mammal.from_pretrained(
-        pretrained_model_name_or_path=os.path.join(
-            finetune_output_dir, "best_epoch.ckpt"
+    if finetune_output_dir:
```
Comment (mosheraboh) [inline mammal/examples/dti_bindingdb_kd/main_infer.py:58]: Can you please add to type annotation None as an option and also add in line 52 explnantion about what will happen if it's None.

File: mammal/examples/dti_bindingdb_kd/main_infer.py [hunk 2]
```diff
@@ -8,7 +8,7 @@
 
 
 @click.command()
-@click.argument("finetune_output_dir")
+@click.argument("finetune_output_dir", default="")
```
Comment (mosheraboh) [inline mammal/examples/dti_bindingdb_kd/main_infer.py:11]: None or empty string?

**Review by mosheraboh — COMMENTED** [review]
Looks great!
Minor comment inline.
Thanks.

File: mammal/examples/dti_bindingdb_kd/main_infer.py (hunk 1 above)
Comment (simona-rc) [reply mammal/examples/dti_bindingdb_kd/main_infer.py]: done

File: mammal/examples/dti_bindingdb_kd/main_infer.py (hunk 2 above)
Comment (simona-rc) [reply mammal/examples/dti_bindingdb_kd/main_infer.py]: Updated to use None instead of empty string

**Review by mosheraboh — APPROVED** [review]
Looks great!

**Pull request merged**